Git.sketchplugin/exportArtboard.sh

```diff
@@ -19,12 +20,21 @@ mv "$FILE_FOLDER" .oldArtboards 2>/dev/null
 IGNORE=$([ -e .sketchignore ] && (cat .sketchignore | sed '/^$/d' | sed 's/^/^/' | sed 's/$/$/' | tr '\n' ',') || echo "")
 
 # get list of artboard names to export
-ARTBOARDS=$($BUNDLE_PATH/Contents/Resources/sketchtool/bin/sketchtool list artboards "$FILENAME" --include-symbols=YES | python "$(dirname "$0")"/getArtboardNames.py "$IGNORE" | tr '\n' ',')
```

Comment:
oops! that is leftover from trying out version 0.7.0 . I tested with 0.8.5 and it works 😄